src/database/utilities/createRecord.js

```diff
@@ -647,6 +647,14 @@ const createItemBatch = (database, item, batchString, supplier) => {
   realItem.addBatch(itemBatch);
   database.save('Item', realItem);
 
+  // If this item batch is for an underlying vaccine item, auto apply a VVM status
+  if (isVaccine) {
+    const vaccineVialMonitorStatus = database
```

Comment:
Does this need any defensive code, e.g.

```
const [vaccineVialMonitorStatus = VACCINE_STATUS_DEFAULT] = database.objects('VaccineVialMonitorStatus').sorted('level')
```

Reply:
implicit applying of falsey value .. maybe it's not great, maybe it's JSaline 🎉 ?